src/Kros.ProblemDetails.Extensions/README.md

```diff
@@ -0,0 +1,34 @@
+# Kros.ProblemDetails.Extensions
+
+**Kros.ProblemDetails.Extensions** je pomocná knižnica, ktorá rozširuje knižnicu [ProblemDetails](https://github.com/khellang/Middleware).
+
+## Extensions
+
+Menný priestor `Kros.ProblemDetails.Extensions` obsahuje rozšírenia pre ProblemDetails.
+
+Registrovaním `services.AddCustomProblemDetails()` pridáte do pipeline-y nastavenie ProblemDetails do HTTP response.
+Registrácia nastaví pre [validačnú chybu](https://github.com/FluentValidation/FluentValidation/blob/master/src/FluentValidation/ValidationException.cs) z fluent validácii response code 400 a podrobnejší ProblemDetail.
+
+```CSharp
+public virtual void ConfigureServices(IServiceCollection services)
+{
+	services.AddCustomProblemDetails();
```

Comment:
Už sa to nevolá `AddCustomProblemDetails`. ;)

Reply:
Upravene